Formula/pumba.rb

```diff
@@ -2,9 +2,10 @@ class Pumba < Formula
   desc "Chaos testing tool for Docker"
   homepage "https://github.com/alexei-led/pumba"
   url "https://github.com/alexei-led/pumba/archive/0.7.7.tar.gz"
-  sha256 "5da828d47d7d46305fc921445ad47d9825a1d54f09b8be8a01ff2095c804fe2d"
+  sha256 "974f17df36bfdbb510b303db7acf1c3d637d7d50c87f3120b07e1e363792cf87"
   license "Apache-2.0"
   head "https://github.com/alexei-led/pumba.git"
+  revision 1
```

Comment:
I added a revision so it will be rebuilt, and reinstalled. As per my investigation, the build process had changed between 0.7.7 releases and it is probably best to ensure the binary is fresh.